dts/arm/nxp/nxp_rw6xx_common.dtsi

```diff
@@ -147,6 +147,14 @@
 	pmu: pmu@31000 {
 		reg = <0x31000 0x130>;
 		compatible = "nxp,rw-pmu";
+		pin0: pin0 {
```

Comment:
set to status disabled and comment that it is not yet enabled if we don't put code for it

Reply:
Done, I have also added support for `pin0` as well and tested with a build locally.